tools/devops/automation/build-cronjob.yml

```diff
@@ -132,7 +132,7 @@ resources:
     checkoutOptions:
       submodules: true
 
-  - repository: templates
+  - repository: xamarin-templates
```

Comment:
Maybe it would be best to use the repository name `yaml-templates` here instead?

Reply:
I'm using the maui name, we will probably want to fix this in the future, but is a good starting point.